### DISCUSSION TITLE
support for new model in tuya.js

### DISCUSSION THREAD

**Discussion by joneskor** [Koenkk/zigbee2mqtt#25073] [Devices]
Hello, 
i'v bought CONECTO ZIGBEE TEMP/HUMIDITY SENSOR (LCD)
model: COZIGTHS1
sold by power.fi
z2m reconizes device as TS0601 / _TZE200_zppcgbdj

i think its rebranded device of    TS0601 / _TZE200_s1xgth2u 
or atleast i got it working by modifying file: ./node_modules/zigbee-herdsman-converters/devices/tuya.js

replaced this line:
        fingerprint: [{ modelID: 'TS0601', manufacturerName: '_TZE200_s1xgth2u' }],

with this:
        fingerprint: tuya.fingerprint('TS0601', [
            '_TZE200_s1xgth2u',
            '_TZE200_zppcgbdj',
        ]),

But im not familiar enough with github to make this modification permanent.
Could someone help with this ?

Also, i have both devices and their picture is wrong, correct picture attached.
![tuya_zigbee_temp_lcd](https://github.com/user-attachments/assets/8860c27e-80e2-4480-8822-dd1bfc521796)

**Comment by Koenkk** [comment]
Added, thanks!

Changes will be available in the [dev branch](https://www.zigbee2mqtt.io/advanced/more/switch-to-dev-branch.html) in a few hours from now. Note that the dev branch currently contains **breaking changes**, see https://github.com/Koenkk/zigbee2mqtt/discussions/24198

**Reply by Koenkk** [reply]
EDIT: Does it not work out of the box with z2m 1.42.0?

**Reply by joneskor** [reply]
actually i was running 1.41.0-dev so cannot tell yet. might test tomorrow if i have time

**Reply by joneskor** [reply]
tested today, and you were right. It works out of the box with 1.4.2 (except battery says null, but that was case with my fix also). sry about this, did not remeber i was on dev branch.
